lib/src/react_client/event_helpers.dart

```diff
@@ -760,44 +760,47 @@ SyntheticWheelEvent createSyntheticWheelEvent({
 }
 
 extension SyntheticEventTypeHelpers on SyntheticEvent {
+  bool _checkEventType(List<String> types) => hasProperty(this, 'type') && types.any((t) => this.type.contains(t));
```

Comment:
#nit it's unlikely, but `this.type` could still technically be `null` here, so perhaps we could combine the `hasProperty` check with a null-check?
```suggestion
  bool _checkEventType(List<String> types) => type != null && types.any((t) => this.type.contains(t));
```